modules/social_features/social_topic/social_topic.install

```diff
@@ -247,3 +247,40 @@ function social_topic_update_8903() {
     }
   }
 }
+
+/**
+ * Updated fieldset labels and added new fieldset called Settings.
+ */
+function social_topic_update_8904() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+
+  // Execute configuration update definitions with logging of success.
+  $updateHelper->executeUpdate('social_topic', 'social_topic_update_8904');
+
+  // Output logged messages to related channel of update execution.
+  return $updateHelper->logger()->output();
+}
+
+/**
+ * Added fields to their new respective fieldsets.
+ */
+function social_topic_update_8905() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+
+  // Execute configuration update definitions with logging of success.
+  $updateHelper->executeUpdate('social_topic', 'social_topic_update_8905');
+
+  // Output logged messages to related channel of update execution.
+  return $updateHelper->logger()->output();
+}
+
+/**
+ * Remove group_topic_description fieldgroup from topic content type.
+ */
+function social_topic_update_8906() {
+  if ($group = field_group_load_field_group('group_topic_description', 'node', 'topic', 'form', 'default')) {
```

Comment:
Would be great if we can add the check we discussed, to see if there are any fields attached to this fieldset. 
Eventhough we move the description out, it could be others have added more fields to this.
We don't want to break it for them, we rather give them the option to move that field through the UI to the new fieldsets if they wish to do so.